root/etc/cont-init.d/98-golang

```diff
@@ -4,7 +4,10 @@ echo "ensuring golang is in PATH"
 if grep -q -E '^(export )?PATH=' /etc/services.d/code-server/run; then
   if ! grep -q -E '^(export )?PATH=.*/usr/local/go/bin.*' /etc/services.d/code-server/run; then
     sed -i '/PATH/ s/$/:\/usr\/local\/go\/bin/' /etc/services.d/code-server/run
+  elif ! grep -q -E '^(export )?PATH=.*/config/go/bin:.*' /etc/services.d/code-server/run; then
+    sed -i 's/PATH=/PATH=\/config\/go\/bin:/g' /etc/services.d/code-server/run
```

Comment:
why not use the same sed method for both?

Reply:
Because the first appends the PATH to the end and the second puts it first. See my previous reply.